citi-onboarding/Recorda-one-page
# Feature/news integration

#What I did
- Made the news integration

#How to test
- `git checkout main`
- `git pull`
- `git checkout feature/news-integration`
- Add some content to the news database and check if it is being shown on frontend
